ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -149,6 +149,39 @@ library
     , these                         >=1.1   && <1.2
     , vector-map
 
+library cardano-legacy-block
+  import:          common-lib
+  hs-source-dirs:  src/cardano-legacy-block
+  exposed-modules:
```

Comment:
Do we need to expose all the modules?

Reply:
I don't see harm in exposing all modules for now